src/Microsoft.Restier.EntityFramework/Model/ModelProducer.cs

```diff
@@ -193,7 +193,7 @@ private ModelProducer()
                         true,
                         out storeGeneratedPattern);
 
-                    if (storeGeneratedPattern != null)
+                    if (storeGeneratedPattern != null && (string)storeGeneratedPattern.Value == "Computed")
```

Comment:
Actually according to OData [standard vocabularies](http://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/vocabularies/Org.OData.Core.V1.xml), we don't support any annotation regarding StoreGeneratedPattern.Identity but only Computed.